### ISSUE TITLE
Should not force <script> to be in <head>

### ISSUE THREAD

**Issue by ford-prefect** [nkunihiko/django-bootstrap3-datetimepicker#17]
It would be good to be able to push all the script bits created by django-bootstrap3-datetimepicker to the end of the file, rather than the beginning.

**Comment by AgustinLado** [comment]
@barchiesi  Indeed! I've got the same problem. It is best to include all the JS after the above-the-fold content, and the datetimepicker templates are in that content.
After over 2 years, is there any way to include the scripts afterwards?
